content/en/blog/k8-basics/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "K8s Basics"
```

Comment:
rename title from`k8-basics` to `kubernetes-basics` as newbie wont be knowing what is k8s mean.